widgetastic_manageiq/__init__.py

```diff
@@ -5141,3 +5141,63 @@ def click_body_action(self):
 
     def click(self):
         return self.click_title()
+
+
+class MultiBoxOrderedSelect(MultiBoxSelect):
+    move_into_button = Button(title=Parameter("@move_into"))
+    move_from_button = Button(title=Parameter("@move_from"))
+    move_up_button = Button(title=Parameter("@move_up"))
+    move_down_button = Button(title=Parameter("@move_down"))
+
+    def __init__(
+        self,
+        parent,
+        available_items,
+        chosen_items,
+        move_into,
+        move_from,
+        move_up,
+        move_down,
+        logger=None,
+    ):
+        MultiBoxSelect.__init__(
+            self,
+            parent=parent,
+            available_items=available_items,
+            chosen_items=chosen_items,
+            move_into=move_into,
+            move_from=move_from,
+            logger=logger,
+        )
+        self.move_up = move_up
+        self.move_down = move_down
+
+    def fill(self, values):
```

Comment:
**Question** Could you use `DefaultFillViewStrategy` here? Define that in the `MultiBoxSelect` class?

Reply:
I think this is required, this (and `MultiBoxSelect`) need to use fill_strategy to define this, as it doesn't handle before_fill/after_fill calls and breaks some expectations of filling widgets.

Reply:
> **Question** Could you use `DefaultFillViewStrategy` here? Define that in the `MultiBoxSelect` class?
@jawatts as per me. when we not specify any `fill_strategy` then `DefaultFillViewStrategy` will select by default. For reference please check `View` from `wt.core`.

Reply:
> I think this is required, this (and `MultiBoxSelect`) need to use fill_strategy to define this, as it doesn't handle before_fill/after_fill calls and breaks some expectations of filling widgets.

@mshriver I am confused about this. I tried to understand but fails as `before_fill` and `after_fill` are hooks; which I am not using. He we are checking order with values not just values. Can you please help me to understand this? I would like to improve this.